posts-api/controllers/post.js

```diff
@@ -100,6 +111,22 @@ exports.getPost = (req, res, next) => {
     });
 };
 
+exports.findPostByServiceName = (req, res, next) => {
```

Comment:
Ovo je deo service descovery postupka
Kada korisnik pogodi rutu /moj-novi-servis, u mongoDb bazi trazimo servis koji ima serviceName 'moj-novi-servis'
Kada nadjemo taj servis onda smo dobili i njegov pravi servicePath (to je ono swapi.dev)